manager/integration/tests/common.py

```diff
@@ -3125,6 +3126,25 @@ def reset_engine_image(client):
     assert ready
 
 
+# ensure that the engine image daemonset scale up for longhorn
+# after scaling down daemonset
+def scale_up_engine_image_daemonset(client):
+    apps_api = get_apps_api_client()
+    default_img = get_default_engine_image(client)
+    ds_name = "engine-image-" + default_img.name
+    body = [{
+        "op": "remove",
+        "path": "/spec/template/spec/nodeSelector/foo"
```

Comment:
So this only removed the node selector `foo`. Is it possible to remove all node selectors?